swc-uxp-starter/src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Not sure whats the purpose of these files under /report. Please include a description in README.